src/pages/HomePage/HomePageComponents/NavBar/NavBar.css

```diff
@@ -0,0 +1,87 @@
+.navigation-container {
+    padding: 1rem 0;
+}
+
+.logo-input {
+    display: flex;
+    width: 70%;
+    margin-left: 1.5rem;
+}
+
+.search-input-container {
+    display: flex;
+    align-items: center;
+    width: 60%;
+    padding: 0 1rem;
+}
+
+.input-icon {
+    position: relative;
+    right: 3rem;
+    padding: 5px;
+}
+
+.header-input {
+    padding: 13px;
+    width: 90%;
+    border-radius: 3rem;
+    border: none;
+}
+
+.basic {
+    border: 0px;
+}
+
+.number-one {
+    background-color: var(--alert-primary);
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    position: relative;
+    left: -0.6rem;
+    margin-right: -1.3rem;
+    bottom: 0.8rem;
```

Comment:
dont use rems in 0. formats

Reply:
Sure, I will correct it and not repeat it again 👍️